pennylane/qnode.py

```diff
@@ -432,6 +447,29 @@ def _validate_device_method(device, interface):
 
         return JacobianTape, interface, device, {"method": "device"}
 
+    @staticmethod
+    def _validate_finitediff_method(device, interface):
+        """Validates whether a JacobianTape interface supports accurate finite differences.
+
+        Args:
+            device (.Device): PennyLane device
+            interface (str): name of the requested interface
+
+        Returns:
+            tuple[.JacobianTape, str, .Device, dict[str, str]]: Tuple containing the compatible
+            JacobianTape, the interface to apply, the device to use, and the method argument
+            to pass to the ``JacobianTape.jacobian`` method.
+        """
+        if interface == "jax":
+            if not _jax_float64_support():
+                warnings.warn(
+                    "float64 support not enabled for jax. "
+                    "May cause inaccuracies for `diff_method='finite-diff'`",
+                    UserWarning,
+                )
+
+        return JacobianTape, interface, device, {"method": "numeric"}

```

Comment:
Yep, I would say this definitely feels like something that lives on the interface rather than the QNode :) Especially because it simply raises a warning, rather than changing the diff method.